templates/focus-ios.xliff

```diff
@@ -28,6 +28,19 @@
       </trans-unit>
     </body>
   </file>
+  <file original="Blockzilla/en.lproj/Intents.strings" source-language="en" datatype="plaintext">
+    <header>
+      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="9.1" build-num="9B55"/>
+    </header>
+    <body>
+      <trans-unit id="Erase">
+        <source>Erase</source>
```

Comment:
Probably better to add a short localization note here to explain what "Erase" refers to exactly (what are we erasing / where?)

Reply:
Whoops, that wasn't supposed to get exported. Sorry about that! New PR: https://github.com/mozilla-l10n/focusios-l10n/pull/60